Framework/DataObjects/src/TimeSplitter.cpp

```diff
@@ -206,8 +206,7 @@ std::string TimeSplitter::getWorkspaceIndexName(const int workspaceIndex, const
 
   // If numericalShift > 0, the caller will get back a shifted index.
```

Comment:
I don't think this comment makes sense, right? Can you adapt it?

Reply:
Done.